### DISCUSSION TITLE
Build warning for font resource size

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#8886] [CLI - PWA mode]
For a while now when building my pwa (with `quasar build -m pwa`), I get this warning at the end:

```
 App · ⚠️  1 warning encountered:
/img/fa-light-300.81ebcae9.svg is 2.44 MB, and won't be precached. Configure maximumFileSizeToCacheInBytes to change this limit.
```

The app still works fine, but I guess I am looking for some advice here:
- Should I change `maximumFileSizeToCacheInBytes`?
- Should I try to reduce the size of fonts files (either by extracting them all to svgs, or something else)?
- Some other action?
- Ignore this warning?

Thanks!

**Comment by hawkeye64** [comment]
@ssuess Why not use the woff/woff2 files instead of the Glyph file? I would (make a copy first) remove all the Glyphs from that file you are not using. The whole thing is getting loaded into the browser.

**Reply by ssuess** [reply]
looking through my setup, I am not entirely sure how I would accomplish that. The fontawesome pro (light) set is being included in the `quasar.conf.js` which references a boot file that seemingly only references the css files (`import '@fortawesome/fontawesome-pro/css/fontawesome.min.css'
import '@fortawesome/fontawesome-pro/css/light.min.css'`). And the `light.min.css` file includes all the regular formats (`eot`,`woff`,`woff2`,`ttf`, and `svg`). Not sure if there is a quasar webpack setting that is reducing everything to svg (I assume so). Separately, there are a few svg glyphs that I have configured manually from other sets, that I import into my components, but only about 5 of them.
